ietf/person/views.py

```diff
@@ -69,11 +69,11 @@ def ajax_select2_search(request, model_name):
 
 
 def profile(request, email_or_name):
-    aliases = Alias.objects.filter(name=email_or_name)
+    aliases = Alias.objects.filter(name__icontains=email_or_name)
```

Comment:
how about iexact instead of icontains in both places?

Reply:
added commit to do this.
(It was useful to each just 'salz'  but not worth the #6008 risk)